packages/core/src/components/alert/alert.tsx

```diff
@@ -10,6 +10,9 @@ export class Alert {
   @Prop()
   status!: 'success' | 'info' | 'warning' | 'danger';
 
+  @Prop()
+  roleAlert?: boolean;
```

Comment:
of, stellen we juist `role` bechikbaar in de API, met vooralsnog als enige mogelijke waarde `'alert'`? wat denk jij @tfrijsewijk

Reply:
Wat zou een toepasselijk andere role zijn voor een Alert wat geen `alert` is ?

Ik vind het geen gek idee, maar is dit niet 'n beetje YAGNI? En misschien ook wel een beetje suf `role-alert="alert"` ?

Reply:
Wat zou een toepasselijk andere role zijn -> geen enkele. Daarom staan we ook alleen `alert` toe. Ik vind het ook stom, maar ik vind het api-technisch toch fraaier. Toegegeven, als er nooit een andere role gaat komen (en dat gaat er niet komen), is een enumeratie met maar 1 mogelijke waarde ook wat stommig, maar toch.

Ik snap je vraag niet over `role-alert="alert"` -> dat is niet wat ik voorstel, ik stel voor `role="alert"` (waarbij alert dus de enige mogelijke waarde is). Eigenlijk dus gewoon precies wat ook in de uiteindelijke HTML gaat komen, maar dan zodanig beperkt dat er alleen `alert` in kan komen.

Reply:
Volgens mij is het niet de bedoeling dat we `role="alert"` op `<dso-alert>` plaatsen. Ik denk dat screenreaders deze namelijk gewoon gaan interpreteren. En aangezien dat de role ook al IN het component op de `<div class="alert">` zetten zal het dubbelop zijn.

Daarnaast vraag ik mij af of we attributen met functionaliteit (zoals `role`) moeten inzetten voor de API van een web component.

Reply:
Nou dat waren dus ook precies de dingen die ik mij afvroeg :) Weet je wat, zullen dan maar gewoon lekker accepteren wat @daverhelst hier heeft geproduceerd?

Reply:
@daverhelst neem je nog wel duidelijk op in de informatie van deze pull-request dat je op het webcomponent dus `role-alert="true"` dient te zetten? Dat blijkt nu slechts impliciet.

Reply:
`role-alert` of `role-alert="true"` ? Voor de tooltip heb ik nu `active` gemaakt, maar ook alleen maar omdat Stencil dat maakt van een property met type `boolean`.. Mogelijk dat beide werken, maar laten we in de documentatie één manier uitdragen. Tooltip is nog niet gemerged, dus we kunnen hier nog een beslissing over nemen

Reply:
Zie aanpassing in openings comment van dit PR